spec/examples/examples.yml

```diff
@@ -1175,9 +1175,9 @@ functions:
           <TabPanel id="js" label="JavaScript">
 
             ```ts
-            const { count, error } = await supabase
-              .from('countries')
-              .select('*', { count: 'exact', head: true })
+            const { error, data, count } = await supabase
+              .from('table')
```

Comment:
Any reason we are no longer referencing the `countries` table?

Reply:
Let's revert back to `countries` here since it is linked to that schema in this example. Thanks!